hack/lib/common.sh

```diff
@@ -380,19 +387,53 @@ EOF
     kube::util::wait_for_url "https://${API_HOST_IP}:$secureport/healthz" "apiserver: " 1 "${WAIT_FOR_URL_API_SERVER}" "${MAX_TIME_FOR_URL_API_SERVER}" \
         || { echo "check apiserver logs: ${APISERVER_LOG}" ; exit 1 ; }
 
-    if [[ "${REUSE_CERTS}" != true ]]; then
+    #if [[ "${REUSE_CERTS}" != true ]]; then
```

Comment:
the REUSE_CERT option should be kept.

Reply:
Scale out case currently runs in non secured mode in local. REUSE_CERTS flag is only used in API server partition case. It consistently having issue with certificates. I am considering only support non secure mode in partitioned API server case.

Reply:
Added comment